### ISSUE TITLE
A11y: All content must reside within a WAI-ARIA landmark or labelled region role.

### ISSUE THREAD

**Issue by xiaolixu** [IBM/carbon-components-react#1238]
## Detailed description

> Describe in detail the issue you're having.
All content must reside within a WAI-ARIA landmark or labelled region role.

===== ENVIRONMENT =====
IBM CI 162 Checkpoint Violation: IBM 2.4.1 Bypass Blocks
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/bypass_blocks.html
Test Tool Type: DAP

===== DEFECT DESCRIPTION =====
All content must reside within a WAI-ARIA landmark or labelled region role.

===== EXPECTED RESULT =====
For more detail:
https://aat.w3ibm.mybluemix.net/token/c3b7a795-a87f-4fe4-8071-059d47e0571f/b5cf2a6d-027b-42c7-aaa0-1e99adf7ac78/latest///doc/w3/help/en-US/idhi_accessibility_check_g1157.html

> Is this a feature request (new component, new icon), a bug, or a general issue?

a11y bug

> Is this issue related to a specific component?

OverflowMenu OverflowMenuItem

> What did you expect to happen? What happened instead? What would you like to see changed?

All content must reside within a WAI-ARIA landmark or labelled region role to facilitate page navigation by screen reader users.

> What browser are you working in?

chrome 68.0.3440.106 (Official Build) (64-bit)
> What version of the Carbon Design System are you using?

`    "carbon-components": "^9.5.0",
    "carbon-components-react": "6.21.4",`
> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
virtual private cloud

## Steps to reproduce the issue

use OverflowMenu with _flipped floatingMenu_

**Comment by joshblack** [comment]
cc @asudoh

**Comment by asudoh** [comment]
Hi @xiaolixu do you mean that the problem doesn’t happen with non-flppped overflow menu? Thanks!

**Comment by xiaolixu** [comment]
@asudoh yeah , it should be specified issue with flipped floatingMenu .

**Comment by asudoh** [comment]
Fiddled with DAP - I see you can make the violation go away by adding `data-floating-menu-container` to `<main>` or an element with an a11y role, e.g.: https://github.com/IBM/carbon-components-react/blob/v6.28.0/.storybook/Container.js#L12-L13

Hope it helps - Feel feel to reopen if you have further questions!
